artemis/modules/admin_panel_login_bruter.py

```diff
@@ -0,0 +1,205 @@
+import logging
+import urllib.parse
+from re import search
+from typing import Optional, Tuple
+
+from karton.core import Task
+from selenium import webdriver
+from selenium.common.exceptions import NoSuchElementException, TimeoutException
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.chrome.service import Service
+from selenium.webdriver.chrome.webdriver import WebDriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.remote.webelement import WebElement
+from selenium.webdriver.support import expected_conditions
+from selenium.webdriver.support.wait import WebDriverWait
+
+from artemis.binds import TaskStatus, TaskType
+from artemis.config import Config
+from artemis.module_base import ArtemisBase
+from artemis.passwords import get_passwords_for_url
+from artemis.utils import remove_standard_ports_from_url
+
+
+class AdminPanelBruterException(Exception):
+    pass
+
+
+class AdminPanelLoginBruter(ArtemisBase):
+    """
+    Tries to log into admin panels with easy passwords (needs "bruter" and "port_scanner" to be enabled as well to find URLs where admin panels reside)
+    """
+
+    identity = "admin_panel_login_bruter"
+    filters = [{"type": TaskType.URL.value}]
+
+    USERNAMES = ["admin"]
+
+    LOGIN_FAILED_MSGS = [
+        "Please enter the correct username and password for a staff account. "
+        "Note that both fields may be case-sensitive.",
+        "Unrecognized username or password. Forgot your password?",
+        "Username and password do not match or you do not have an account yet.",
+        "Invalid credentials",
+        # rate limit
+        "failed login attempts for this account",
+        # pl_PL
+        "Podano błędne dane logowania",
+        "Nieprawidłowa nazwa użytkownika lub hasło",
+        "Złe hasło",
+        "Błędna nazwa użytkownika",
+    ]
+
+    def run(self, task: Task) -> None:
+        url = task.get_payload(TaskType.URL)
+        url_parsed = urllib.parse.urlparse(url)
+
+        is_root_url = url_parsed.path in ["", "/"]
+
+        if not is_root_url and not any([item in url.lower() for item in ["login", "admin", "cms", "backend", "panel"]]):
+            self.db.save_task_result(
+                task=task,
+                status=TaskStatus.OK,
+                status_reason=f"URL {url} doesn't look like an admin panel URL",
+                data=None,
+            )
+            return
+
+        if "admin" in url and not url.endswith("/admin") and "admin" in task.get_payload("found_urls"):
```

Comment:
I don't get this check and task result reason doesn't give clear indication what it is about